From: =?utf-8?q?=22IOhannes_m_zm=C3=B6lnig_=28Debian/GNU=29=22?=
 <umlaeute@debian.org>
Date: Tue, 23 Oct 2018 21:49:33 +0200
Subject: exclude stripped sources from build

since we stripped out the entire tests/ directory, we must not include it in
the build
Forwarded: not-needed (Debian specific)
---
 code/D3MFImporter.cpp           | 2 +-
 code/D3MFOpcPackage.cpp         | 2 +-
 code/IFCLoader.cpp              | 2 +-
 code/Q3BSPZipArchive.h          | 2 +-
 tools/assimp_cmd/CMakeLists.txt | 1 -
 5 files changed, 4 insertions(+), 5 deletions(-)

--- assimp.orig/tools/assimp_cmd/CMakeLists.txt
+++ assimp/tools/assimp_cmd/CMakeLists.txt
@@ -48,7 +48,6 @@
 LINK_DIRECTORIES( ${Assimp_BINARY_DIR} ${Assimp_BINARY_DIR}/lib )
 
 ADD_EXECUTABLE( assimp_cmd
-  assimp_cmd.rc
   CompareDump.cpp
   ImageExtractor.cpp
   Main.cpp
